### PR TITLE
programme that says if a number is prime

### PR DESCRIPTION
**Pull request by RubenArjona**

### CODE REVIEW

**Comment by atakan** [comment]
I'd like to review this.

File: Arjona/template/simple_functions.py
Comment (atakan) [inline Arjona/template/simple_functions.py:15]: It would be more useful if this was a function, rather than a standalone program.

File: Arjona/template/simple_functions.py
Comment (atakan) [inline Arjona/template/simple_functions.py:24]: Probably sqrt(n) is more useful than n//2.

File: Arjona/template/simple_functions.py
Comment (atakan) [inline Arjona/template/simple_functions.py:35]: If this was a function, these print() statements would need to be changed into returning True or False values.

**Review by atakan — CHANGES_REQUESTED** [review]
Generally sound approach. A few suggestions below.

**Comment by bsipocz** [comment]
@RubenArjona @atakan - Thanks for opening the pull request and for the review(s)! Below I have provided a few comments about the *process* of the pull request and review. Note that we are not conducting a review of the code itself, because the exercise was about the overall process not the details of the code. Also, because there are pending suggestions, we will not merge this PR until the end of Astro Hack Week.

@RubenArjona - it would have been good to provide a description for the PR, even if it duplicates a little the title.

@atakan - you did not "approve" or "request changes" when you submitted the review.  Without that a maintainer isn't sure if your review should be taken as a complete review or just general comments.

**Comment by bsipocz** [comment]
As a closure to AstroHackWeek, we go ahead and merge this now. Thank you both.

**Pull request merged**
